### ISSUE TITLE
Integration request: iCloud (Notes)

### ISSUE THREAD

**Issue by dot-Justin** [dot-Justin/LAMAtHome#1]
**Describe the integration you'd like**
- Allow sign into iCloud, create new notes

**Potential syntax for your integration**
- "iCloud [Function]"
- "iCloud Note Hello world"

**Website URL to automate (or thing to be locally automated)**
- https://www.icloud.com/

**Additional context**
- none

**Comment by glovergaytan-fs** [comment]
I don't think there is a smooth experience way to get notes, icloud requires a code from your device on every login and that breaks the experience. 

However, I may be able to make calendar and reminders to work. I think this one can be closed unfortunately. 

If anyone has any ideas for a smoother experience, please let me know.

**Comment by dot-Justin** [comment]
Yes we can close this and come back to it later if needed @glovergaytan-fs
